src/views/Graph.vue

```diff
@@ -526,6 +559,27 @@ export default {
           ]
         `)
       }
+
+      if (this.groupCyclePoint) {
+        // Loop over the subgraphs
+        Object.keys(cycleSubgraph).forEach((key, i) => {
+          // Loop over the nodes that are included in the subraph
+          const nodeFormattedArray = cycleSubgraph[key].map(a => `"${a.id}"`)
+          ret.push(`
+          subgraph cluster_margin
+          {
+            margin=50.0
+            label="margin"
+            subgraph cluster_${i} {${nodeFormattedArray};\n
+              label = "${key}";\n
```

Comment:
Could do with upping the font-size of the label.

The label size can be defined in graphviz like this:

```
              label = "${key}";\n
              fontsize = 40
```

Will need to get the `GraphSubgraph` label font size to match this.